src/content.js

```diff
@@ -29,6 +29,9 @@ function adjustDisplayNameColor() {
 function extractBackURL() {
   let swLink = document.querySelector('#menu--account *[data-testid="awsc-switch-roles"]');
   if (swLink) return new URL(swLink.href);
+  
+  let swLink = document.querySelector('#menu--account *[data-testid="awsc-switch-role"]');
```

Comment:
Hi @ryan-shaw , this will result in `Uncaught SyntaxError: Identifier 'swLink' has already been declared`. Thanks @ajchili for catching this!

A simple update of the selector to: `#menu--account *[data-testid^="awsc-switch-role"]` above will allow this to continue to work as this change roles out across the console. Note the `^=` which is "begins with", therefore matching both `awsc-switch-role` and `awsc-switch-roles`

Reply:
Thanks! I have updated this.